crypto/sig/README.md

```diff
@@ -77,10 +75,13 @@ The [ed25519 manuscript]((http://ed25519.cr.yp.to/ed25519-20110926.pdf)) from 20
 > a cipher such as AES, combined with standard PRF-stretching mechanisms to support a long input;
 > but we prefer to reuse `H` to save area in hardware implementations.
 
-We point out that Blake was one of the SHA-3 finalists, and was also explicitly designed to model a PRF.
-We prefer Blake2b here because it reduces the number of different hash functions in our system overall.
+We point out that Merlin Transcripts implement a subset of the STROBE protocol, which states in [STROBE Protocol Framework](https://strobe.sourceforge.io/),
 
-Assuming the Blake2b has the secret-prefix-PRF property, we can say that the signatures created this way
+>  Strobe is based on SHA-3, or rather Keccak-f and cSHAKE (draft NIST SP 800-185).
+
+We prefer using Merlin Transcripts here for simplicity, as we are constructing two transcripts in the course of constructing the signature, and because it reduces the number of different hash functions in our system overall.
```

Comment:
suggestion:

"We prefer using Merlin transcripts to produce the nonce here, instead of a cryptographic hash function. Schnorrkel already relies on Merlin -- it has several benefits, like automatic domain separation and framing. By using this here instead of a hash function like SHA3, we can reduce the total number of cryptographic assumptions underpinning this signature scheme."